modyn/config/schema/pipeline.py

```diff
@@ -429,6 +429,9 @@ class TimeTriggerConfig(BaseModel):
         description="Interval length for the trigger as an integer followed by a time unit: s, m, h, d, w, y",
         pattern=rf"^\d+{_REGEX_TIME_UNIT}$",
     )
+    sample_size: int | None = Field(None, description="The number of samples used for the metric calculation.", ge=1)
+    metric: str = Field("model", description="The metric used for drift detection.")
+    metric_config: dict[str, Any] = Field(default_factory=dict, description="Configuration for the evidently metric.")
```

Comment:
I don't understand these changes - why do they belong to the time trigger?

Reply:
Oh yeah, that should not be here. It probably chose the wrong variant when resolving a merge conflict.